byron-proxy/src/exec/Logging.hs

```diff
@@ -81,6 +81,8 @@ defaultLoggerConfig = Monitoring.Representation
   , Monitoring.hasPrometheus   = Nothing
   , Monitoring.hasGUI          = Nothing
   , Monitoring.options         = mempty
+  , Monitoring.hasGraylog      = Nothing
+  , Monitoring.logOutput       = Nothing
```

Comment:
Apparently a new build of iohk-monitoring has been brought in recently, and bryon-proxy wasn't updated. How did this pass CI?

Reply:
@jbgi could you take a look why this slipped through CI